react/react-training/src/app/components/AppRoutes/components/AuthRoute/index.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { Redirect, Route } from 'react-router';
+import PropTypes from 'prop-types';
+
+export default class AuthComponent extends Component {
```

Comment:
`export default ..` always to the bottom of the file. Also, this comp could be a functional comp